### PR TITLE
Fix footer logo overlapping screen content

### PR DESCRIPTION
**Pull request by kopijunkie**
# Problem
SDK footer, i.e. the "Powered by Onfido" line, overlaps the content of some screens - specifically the Uploader and Welcome screens. 

This also fixes the UI regression reported in CX-3902 where the SDK is not centered in IE11.

# Solution
Remove/re-factor inline styles intended for demo app when in Previewer view/mode to not interfere with existing SDK styles.

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [ ] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [ ] Have new automated tests been implemented?
- [ ] Have new manual tests been written down?
- [ ] Have tests passed locally?
- [ ] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-755-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-755-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-755-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-755-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://755-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-755-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-755-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://755-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by bartzet** [comment]
image

**Comment by kopijunkie** [comment]
> Regarding CX-3902 I think we have introduced the issue that the SDK is glued to the top of the window.
> 
> See screenshot above.

Have pushed the fix for this issue

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-755-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-755-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://755-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-755-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-755-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://755-pr-onfido-sdk-ui-onfido.surge.sh

File: src/demo/demo.ejs [hunk 1]
```diff
@@ -19,29 +19,28 @@
...
         width: 100%;
         height: 100%;
-        display: flex;
-        align-items: center;
-        justify-content: center;
+        margin: 0;
+        padding: 0;
       }
       #demo-app {
         position: relative;
```
Comment (stefaniacardenas) [inline src/demo/demo.ejs]: I think you want to remove this double `position: relative` here

Comment (stefaniacardenas) [reply src/demo/demo.ejs]: It causes the SDK to disappear when you squeeze the window

Comment (kopijunkie) [reply src/demo/demo.ejs]: Good spot! 😱 
Have fixed it and removed the duplicated `position: relative`

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-755-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://755-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-755-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-755-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-755-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://755-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-755-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-755-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://755-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**